### DISCUSSION TITLE
Fantastic library! Got reveresed RGB on ST7789, but easy to resolve.

### DISCUSSION THREAD

**Discussion by mathos00** [moononournation/Arduino_GFX#202] [General]
Been using the Adafruit GFX on a few projects, mostly ST7789. This GFX is MUCH faster. My ESP32 benchmark is basically the same as posted here.

Anyway, for some reason the RGB was reversed, which is strangely not the case in the Adafruit GFX version.

After some research, easily resolved with gfx->invertDisplay(1), just in case other newbies come across the same issue.

Thanks for the great work!

**Comment by moononournation** [comment]
IPS parameter will do the job
